apps/yggdrasil/test/test_helper.exs

```diff
@@ -1,4 +1,8 @@
 ExUnit.start()
 
-Mix.Task.run "ecto.create", ~w(-r Yggdrasil.Repo --quiet)
-Mix.Task.run "ecto.migrate", ~w(-r Yggdrasil.Repo --quiet)
+repo = ~w(-r Yggdrasil.Repo --quiet)
+
+Mix.Task.run "ecto.drop", repo
+Mix.Task.run "ecto.create", repo
+Mix.Task.run "ecto.migrate", repo
+Mix.Task.run "run", ~w(apps/yggdrasil/priv/repo/seeds.exs)
```

Comment:
We can't migrate locally because of the seed script? Could it test for existing data to allow it to be re-runnable? Can a migration script add data?

I know at work we write deltas for static data or data migrations and eventually rescript the database and remove deltas. All deltas have to be re-runnable for us.